src/shared/utils/number-to-US-locale.test.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* global describe, test, expect */
+import numberToUSLocale from './number-to-US-locale'
+
+describe('numberToUSLocale', () => {
+  test('should return the original value if isNaN(value) is true', () => {
+    // Given
+    const value = null
+
+    // When
+    const returnValue = numberToUSLocale(value)
+
+    // Then
+    expect(returnValue).toBe(value)
+  })
+  test('should return a thousands comma separated number if isNaN(value) is false', () => {
+    let value, returnValue
+    // Given
+    value = '123456789'
+
+    // When
+    returnValue = numberToUSLocale(value)
+
+    // Then
+    expect(returnValue).toBe('123,456,789')
+
+    // Given
+    value = 987654312345
+
+    // When
+    returnValue = numberToUSLocale(value)
+
+    // Then
+    expect(returnValue).toBe('987,654,312,345')
+  })
```

Comment:
Great to se tests!
I think it could be useful to add tests for numbers that you don't expect to be changed as well, like `1` and `100`.
If we ever change the implementation we know we're not introducing a regression on on-formatted numbers.